doc/source/tutorials/animations.rst

```diff
@@ -0,0 +1,150 @@
+.. _animations:
+
+.. index:: animations
+
+How to create an animation using Emperor
```

Comment:
Is this a question or will it be better to say: "Creating an animation using Emperor"?